ml_data_management_code_NB.Rmd

```diff
@@ -16,1892 +16,28 @@ library(psych) # to analyze data and descriptive statistics
 library(pander) # to make some clean tables
 library(Rmisc) # to use summarySE() function
 library(gtsummary) # to make some cool summary tables that fit APA format
+library(here) # to make it easier to grab files from the folder where this one sits
 
 ```
 
 ## Load in RScript where functions are created
 
-``` {r include = TRUE}
-
-#source(__.R)
-
-```
-
-## Download Simulated Data
+We don't have an RScript with functions yet
```

Comment:
This is the type of thing I would not commit. You aren't ready for a pull request yet. So, I would be more inclined to wait until you have a meaningful change before creating a PR. In other words, not all incremental changes have to go in the PR. That being said, sometimes it is useful, so I'll defer to you in this particular case.